### ISSUE TITLE
How can i run this on a cgroups2 host?

### ISSUE THREAD

**Issue by tilllt** [oposs/tl-docker#6]
Well, the title says it all, how can i run this on a cgroups2 host?

**Comment by oetiker** [comment]
I found that running systemd on docker has become rather problematic while things worked out of the box for me with podman. so my suggestion: try starting the docker with podman ... it might just work.

**Comment by tilllt** [comment]
The machine I need to be using is running an entire ecosystem based on docker so switching to podman is not unfortunately not an option.

I tried podman in the past and it had some nasty bugs, bad documentation and -  in comparison - a very small community, maybe that has changed now but it was not on par with docker.

**Comment by oetiker** [comment]
Hi @tilllt 

by all means, if you have ideas on how to get a systemd based container running on current docker and kernel in rootless mode, I would be very interested in you help! I already spent hours on trying to get that to work :(

**Comment by tilllt** [comment]
Let me rephrase: are there potential problems that can be caused by keeping the existing docker environment and run the thinlinc container via podman side by side? I just don't want to convert the whole box to podman for the aforementioned reasons.
